Makefile

```diff
@@ -129,7 +129,7 @@ test-unit: ## Run unit tests
 test-integration: $(GO_BIN)/commander ## Run integration tests. To run a specific test pass the FILTER var. Usage: `make test-integration FILTER="attack protection"`
 	${call print, "Running integration tests"}
 	@$(MAKE) install # ensure fresh install prior to running test
-	auth0 tenants add ${AUTH0_CLI_CLIENT_DOMAIN} --client-id ${AUTH0_CLI_CLIENT_ID} --client-secret ${AUTH0_CLI_CLIENT_SECRET}  && commander test ./test/integration/test-cases.yaml --filter "$(FILTER)"; \
+	auth0 login --as-machine --domain ${AUTH0_CLI_CLIENT_DOMAIN} --client-id ${AUTH0_CLI_CLIENT_ID} --client-secret ${AUTH0_CLI_CLIENT_SECRET}  && commander test ./test/integration/test-cases.yaml --filter "$(FILTER)"; \
```

Comment:
Do we need `--as-machine` at all, if the user can simply pass the needed values through flags?